.github/workflows/ci-uto.yaml

```diff
@@ -0,0 +1,189 @@
+name: Build and test Universal Transfer Operator
+on:
+  push:
+    branches: [ 'main', 'release-**' ]
+    paths:
+      - 'python-sdk/**'
```

Comment:
Why do we want to trigger the UTO tests when there is a change to the python-sdk?

Reply:
Hmm...yes should remove it. I copied this file from SQL-CLI. I guess we can remove it from there as well right? 

https://github.com/astronomer/astro-sdk/blob/361031295b4eec77cd6de0607b8648968a8c3c6c/.github/workflows/ci-sql-cli.yaml#L6